src/js/modules/articles/actions.js

```diff
@@ -1,11 +1,67 @@
 import axios from 'axios';
 import * as t from './actionTypes';
+import { getSections } from '../sections/selectors';
 
-const apiUrl = 'http://api.icndb.com/jokes/random';
+const apiUrl = 'https://api.stuyspec.xyz/articles';

```

Comment:
Write a `src/js/constants.js` file that exports a url, `export const API_URL = ...`, that you import back into this actions file. When you fetch, use string interpolation to combine that imported constant with `/articles`.

Just so you know, it's convention for constants to be completely uppercase, constants like an API url, width and height of a screen, etc.